src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php

```diff
@@ -369,7 +369,7 @@ protected function buildFilterParamsFor(\DC_Table $dataContainer, $elementName)
      */
```

Comment:
The `@return string` should now be changed to `@return string|null`

Reply:
done